infrastructure/certificates/tickets.gov.bc.ca.conf

```diff
@@ -3,9 +3,13 @@
 #
 # Generate CSR
 #   openssl req -new -out tickets.gov.bc.ca.csr -newkey rsa:2048 -nodes -sha256 -keyout tickets.gov.bc.ca.key -config tickets.gov.bc.ca.conf
+#   or using docker
+#   docker run -it --rm -v %cd%:/apps -w /apps alpine/openssl req -new -out tickets.gov.bc.ca.csr -newkey rsa:2048 -nodes -sha256 -keyout tickets.gov.bc.ca.key -config tickets.gov.bc.ca.conf
 #
 # Verify CSR
 #   openssl req -test -noout -verify -in tickets.gov.bc.ca.csr 
```

Comment:
Note: -test gives me an error - I think this command is invalid.  Using "-text" does seem to work, however.
```
req: Unrecognized flag test
req: Use -help for summary.
```

Reply:
Fixed the -test typo.